packages/rocketchat-grant/server/index.js

```diff
@@ -12,9 +12,33 @@ import Providers, { middleware as providers } from './providers';
 import Settings from './settings';
 
 let grant;
+let storeAddress = process.env.MONGO_URL || 'mongodb://localhost:27017/';
+
+if (storeAddress.indexOf('connectTimeoutMS=') < 0) {
+	if (storeAddress.indexOf('?') < 0) {
+		storeAddress += '?connectTimeoutMS=1200';
+	} else {
+		storeAddress += '&connectTimeoutMS=1200';
+	}
+}
+const SessionStorage = require('connect-mongodb-session')(session);
+const store = new SessionStorage({
+	collection: process.env.SESSION_COLLECTION || 'rocketchat_grant_sessions',
+	databaseName: process.env.SESSION_DATABASE || 'rocketchat',
```

Comment:
Would this force the `rocketchat` database instead of getting from connection URI?